go/teams/teams.go

```diff
@@ -2677,7 +2677,8 @@ func (t *Team) notify(ctx context.Context, changes keybase1.TeamChangeSet, lates
 	if latestSeqno > 0 {
 		err = HintLatestSeqno(m, t.ID, latestSeqno)
 	}
-	t.G().NotifyRouter.HandleTeamChangedByBothKeys(ctx, t.ID, t.Name().String(), t.NextSeqno(), t.IsImplicit(), changes, keybase1.Seqno(0), keybase1.Seqno(0))
```

Comment:
turned out the real bug was here - passing `t.NextSeqno()` as `latestSeqno` was feeding frontend seqno that was off by one, and when the real notification with seqno+1 came, it was ignoring it.

Reply:
LGTM